capstone/src/main/java/com/google/univiz/UniversityData.java

```diff
@@ -0,0 +1,67 @@
+package com.google.univiz;
+
+import com.google.auto.value.AutoValue;
+
+@AutoValue
+public abstract class UniversityData {
+
+  public static UniversityData create(
+      String name,
+      String city,
+      float latitude,
+      float longitude,
+      int flagMainCampus,
+      int urbanizationDegree,
+      float admissionRate,
+      float avgSAT,
```

Comment:
per style, please use [lower camel case](https://google.github.io/styleguide/javaguide.html#s5.3-camel-case) (and in the accessor method, as well)